src/styles/blocks/header.scss

```diff
@@ -0,0 +1,20 @@
+.header {
+  box-shadow: 0 2px 4px 0 #0000000d;
+
+  @include font;
+
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+
+  padding: 0 50px;
+
+  &__logo {
+    display: flex;
+    padding-right: 243px;
```

Comment:
Using a fixed padding-right value of `243px` might cause layout issues on different screen sizes. Consider using relative units or media queries to adjust the padding for better responsiveness.